### PR TITLE
Feature routing

### PR DESCRIPTION
**Pull request by brdeju**

### CODE REVIEW

File: app/partials/dashboard/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<div>
```
Comment (maniolek) [inline app/partials/dashboard/dashboard.html:1]: What about dashboard? It's ready to implement routing

File: app/js/app/controllers/DashboardCtrl.js [hunk 2]
```diff
@@ -26,6 +27,7 @@
             var $scope = this.$scope;
             var notification = this.notification;
             var resource = this.resource;
+            var dataStorage = this.dataStorage;
             $scope.test = 'asd';
 
             $scope.tesparam = 'aa';
```
Comment (maniolek) [inline app/js/app/controllers/DashboardCtrl.js:33]: ?

File: app/js/app/controllers/DashboardCtrl.js [hunk 3]
```diff
@@ -26,6 +27,7 @@
             var $scope = this.$scope;
             var notification = this.notification;
             var resource = this.resource;
+            var dataStorage = this.dataStorage;
             $scope.test = 'asd';
```
Comment (maniolek) [inline app/js/app/controllers/DashboardCtrl.js:31]: ?

File: app/js/app/configs/Init.js [hunk 4]
```diff
@@ -3,12 +3,35 @@
...
                 var config = {};
                 _.each($rootElement[0].attributes, function(attr) {
                     Config[attr.name.replace('data-', '')] = attr.value;
                 });
+                
+                $rootScope.$on('$routeChangeStart', function (event, next) {
+                    var authorised;
+                    if (next.access !== undefined) {
+                        authorised = AuthProvider.isLoggedIn(); 
+                        console.log( "authorised: " + authorised );
```
Comment (maniolek) [inline app/js/app/configs/Init.js]: if it works, remove pls

File: app/js/app/configs/Init.js [hunk 5]
```diff
@@ -3,12 +3,35 @@
...
+                        var route = next.$$route.originalPath.replace('/','');
+                        console.log( route );
```
Comment (maniolek) [inline app/js/app/configs/Init.js]: same as above comment

File: app/js/app/controllers/LoginCtrl.js [hunk 6]
```diff
@@ -36,12 +36,19 @@
...
-            /**
-             * True = login form, False = register form
-             * @type {boolean}
-             */
-            $scope.form = true;
-
+            $scope.goTo = function(event,route) {
+                event.stopPropagation();
+                event.preventDefault();
+                console.log( route );
```
Comment (maniolek) [inline app/js/app/controllers/LoginCtrl.js]: remove if all works.

File: app/js/app/controllers/LoginCtrl.js [hunk 7]
```diff
@@ -36,12 +36,19 @@
             $scope.loginData = {};
```
Comment (maniolek) [inline app/js/app/controllers/LoginCtrl.js:36]: loginData and registerData can be replaced by one variable when you using goTo function. Keep in mind to clear it after each invoke of that method.

File: public/index.html [hunk 8]
```diff
@@ -31,7 +32,7 @@
 
 <body>
 
-    <div class="container" data-monitool-app="app" data-root-api="http://monitool-gui.github/" ng-controller="DashboardCtrl">
+<!--    <div class="container" data-monitool-app="app" data-root-api="http://monitool-gui.github/" ng-controller="DashboardCtrl">
```
Comment (maniolek) [inline public/index.html]: Remove pls

File: app/partials/dashboard/dashboard.html (hunk 1 above)
Comment (brdeju) [inline app/partials/dashboard/dashboard.html:1]: dashboard has not been merged to development yet, after this merge i merge development branch to dashboard (I wrote you about this..)

**Pull request merged**